modules/ROOT/pages/access-source-message-attributes.adoc

```diff
@@ -0,0 +1,146 @@
+= Accessing Source Message Attributes in Translation Maps
+
+*Vijay, I am finding the text describing the use cases awkward when I use the term "accessing." To me, you are adding source attributes to translation maps or maybe exposing them, but not really accessing them. If you agree with "Exposing Source Message Attributes in Translation Maps,"" I will change the use case wording accordingly.* 
```

Comment:
Hi Diane - I believe we are aligned on keeping this as "Accessing source message attributes" per our discussion earlier.

Reply:
OK